google/resource_kms_key_ring.go

```diff
@@ -79,7 +80,7 @@ func resourceKmsKeyRingCreate(d *schema.ResourceData, meta interface{}) error {
 
 	log.Printf("[DEBUG] Created KeyRing %s", keyRing.Name)
 
-	d.SetId(keyRingId.terraformId())
+	d.SetId(keyRingId.keyRingId())
```

Comment:
Is #terraformId() now unused? If so, we should remove it

Reply:
Done.

Reply:
Ah, got it - just convenience in the key tests.  Since we have to build up the key ring before we build the key, and since the key's ID isn't known back in the test method, it's just easier to call terraformId() on the key ring then add a slash.